shops/templates/shops/weekly_imports_all.html

```diff
@@ -1,6 +1,49 @@
-<html>
+{% extends "shops/base.html" %}
+{% block title %}Weekly shop details{% endblock %}
 
+{% block body %}
+    <label>Select Shop<label>
+        <script src="http://127.0.0.1:8000/static/shops/js/htmx.min.js" crossorigin="anonymous"></script>
```

Comment:
This will break as soon as you deploy your code (e.g. to a server). 

Use {% static ... instead. See https://docs.djangoproject.com/en/4.1/intro/tutorial06/#customize-your-app-s-look-and-feel